### ISSUE TITLE
Why did you start to call component architecture as "microfrontends"?

### ISSUE THREAD

**Issue by f1ames0ff** [neuland/micro-frontends#57]
The topic.

**Comment by stevula** [comment]
I agree. The page seems to be more about Web Components than micro-frontends as a general concept. I find [the article from ThoughWorks](https://martinfowler.com/articles/micro-frontends.html) gives a more thorough summary of some other approaches besides web components.

**Comment by nedredmond** [comment]
It's a decent primer.

**Comment by sirugh** [comment]
Resurrecting an old post. Oh well.

I feel that the term "micro frontends" (MFEs), as used in both this site and the Martin Fowler post, are just describing two different approaches to front end development as it correlates to the organizational structures and responsibilities of the engineering teams working on the application. They literally start this post detailing a "vertical" organization.

Call it what you will, but I think it's perfectly acceptable to try and define or generalize something like scalable front-end architecture, even if it doesn't align 1:1 with some other opinion about the subject.

**Comment by wwmike** [comment]
For me as a backend focused fullstack developer it was an amazing reading. So, thank you very much!

( I'm always  a bit sad when I read issues like this, where people are just trying to look smart; while not even bothering to say thank you for  receiving someone's hard work for free. )

**Comment by brent-hifin** [comment]
For what it's worth, [Building Microservices](https://a.co/d/0ObPztR) refers to this pattern as "widgets" - which are front end web applications that can exchange information.
